pkg/sql/opt/optbuilder/export.go

```diff
@@ -31,6 +33,28 @@ func (b *Builder) buildExport(export *tree.Export, inScope *scope) (outScope *sc
 	fileName := b.buildScalar(
 		texpr, emptyScope, nil /* outScope */, nil /* outCol */, nil, /* colRefs */
 	)
+
+	admin, err := b.catalog.HasAdminRole(b.ctx)
+	if err != nil {
+		panic(err)
+	}
+	if !admin {
+		destinationURI, err := texpr.Eval(b.evalCtx)
+		if err != nil {
+			panic(err)
+		}
+		dest := strings.Trim(destinationURI.String(), `'"`)
```

Comment:
What's this for?

Should this be MustBeDString or AsString instead of String() ?

Reply:
resolved.